lc0_main.go

```diff
@@ -59,6 +62,7 @@ var (
 	keep          = flag.Bool("keep", false, "Do not delete old network files")
 	version       = flag.Bool("version", false, "Print version and exit.")
 	trainOnly     = flag.Bool("train-only", false, "Do not play match games")
+	stats         = flag.Bool("stats", false, "Send system info to server for more fine-grained statistics")
```

Comment:
I think it might be better to split hostname/gpu number to be conditional on a separate flag.  Gpu type seems something we might be be able to convince more people to enable than hostname.

Reply:
Now there are two flags, `report-gpu` and `report-host`.